app/controllers/tweets_controller.rb

```diff
@@ -1,13 +1,15 @@
 class TweetsController < ApplicationController
   def new
   end
+
   def create
-    @tweet = Tweet.new(user_id: current_user.id, text: params[:text])
+    @tweet = current_user.tweets.build(user_id: current_user.id, text: params[:text])
```

Comment:
current_user.tweets.buildの時点でuser_idにあらかじめcurrent_user.idが設定されているので、
user_id: current_user.idは不要です。
```suggestion
    @tweet = current_user.tweets.build(text: params[:text])
```

Reply:
params[:text]を直接渡すのではなくstrong parameterを使いましょう。

```suggestion
    @tweet = current_user.tweets.build(tweet_params)
    
    
    private
    
    def tweet_params
      params.require(:tweet).permit(:text)
    end
```

上記で動くようにparams[:tweet][:text])でPOSTされるようにviewを修正しましょう。